src/components/HomePage/SearchArea/ShowAdvancedOptionsButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Fab from '@material-ui/core/Fab';
+import Close from '@material-ui/icons/CloseRounded';
+import MoreHoriz from '@material-ui/icons/MoreHoriz';
+
+import searchAreaStyle from "./SearchArea.module.css";
+
+const ShowAdvancedOptionsButton = ({open, onClick}) => {
+    return (
+        <div className={searchAreaStyle.advancedSearchBtnWrapper}>
+            <Fab
+                color="primary"
+                aria-label="Show More Options"
+                onClick={onClick}
+            >
+                {open ? <Close /> : <MoreHoriz fontSize="large"/>}
```

Comment:
Would kinda expect a Close/Open pair but can understand the usage of MoreHoriz. Could/Should be debated

Reply:
The thoughts were the following:
- There is already a chevron to see more of the page
- Since the inputs are reset when the advanced are is close, the user should know the action is to "cancel" rather than closing the section, hence the cross instead of the previous chevron-up

Reply:
Last time I checked it was a chevron-up, hence the comment, my bad for not checking! Agree, 🚢 📦  -> :shipit: 
Resolving